### ISSUE TITLE
Consider hosting databases/datasets on alternative provider

### ISSUE THREAD

**Issue by funnell** [bhattlab/phanta#7]
Hello! I'm excited to try out phanta, but unfortunately my institution has blocked all access to Dropbox, so I'm unable to download the databases or test dataset.

Would you be able to host the data on an additional provider? I suspect I won't be the only one with blocked access to Dropbox...

Thank you very much for your help!

**Comment by meenachakra** [comment]
Thank you for the comment! We hear you, we totally agree and plan to release a version on an alternative provider within a few weeks. Thanks for your interest in Phanta and you will get to use it soon!

**Comment by nickp60** [comment]
Hi, I saw this issue was closed: is there a new download link?
Also, is there a timeline for when the masked db will be available?

Thanks in advance!

**Comment by meenachakra** [comment]
Hi Nick,
Thank you! There is not a new download link yet. Two updates for you: 1) we are actively trying to get your version of the pipeline (with std resource names) working on our cluster - we've contacted our cluster admins, 2) we closed the issue because we similarly have plans to release both the masked and unmasked DB on an alternative provider after Thanksgiving. Also tagging @yipinto. Thanks!
